backends-common/cassandra/pom.xml

```diff
@@ -192,26 +192,6 @@
                 </plugins>
             </build>
         </profile>
-        <profile>
-            <id>disable-animal-sniffer</id>
-            <activation>
-                <jdk>[1.6,)</jdk>
-            </activation>
-            <build>
-                <plugins>
-                    <plugin>
-                        <groupId>org.codehaus.mojo</groupId>
-                        <artifactId>animal-sniffer-maven-plugin</artifactId>
-                        <executions>
-                            <execution>
-                                <id>check_java_6</id>
-                                <phase>none</phase>
-                            </execution>
-                        </executions>
-                    </plugin>
-                </plugins>
-            </build>
-        </profile>
```

Comment:
I disagree. We should active animal-sniffer everywhere, but it is desactivated on some projects because it is buggy.
We should check if the bug is fixed and then reactivate it on all projects.